TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Subsystems/Claw.java

```diff
@@ -1,22 +1,29 @@
 package org.firstinspires.ftc.teamcode.Subsystems;
 
+import com.arcrobotics.ftclib.command.SubsystemBase;
 import com.qualcomm.robotcore.hardware.DcMotor;
 import com.qualcomm.robotcore.hardware.Servo;
 import com.qualcomm.robotcore.hardware.HardwareMap;
 
-public class Claw {
+public class Claw extends SubsystemBase {
 
     // Motor Declaration
     private Servo grab_RightServo; // Declare left claw servo
     private Servo grab_LeftServo; //Declare right claw servo
+    private Servo twist_Right; //Declare right arm
+    private Servo twist_Left; //Declare left arm
 
     public Claw (HardwareMap hardwareMap) {
         // Servos IDs
         grab_RightServo = hardwareMap.get(Servo.class, "grab_RightServo");
         grab_LeftServo = hardwareMap.get(Servo.class, "grab_LeftServo");
+        twist_Right = hardwareMap.get(Servo.class, "twist_Right");
+        twist_Left = hardwareMap.get(Servo.class, "twist_Left");
         // Reverse the reversed Servo
         grab_RightServo.setDirection(Servo.Direction.FORWARD);
```

Comment:
but why

Reply:
im sorry but here is correct no problem theres 4 servos left rigth twist left right grab